sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/view/AttributesProcessor.java

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.sdk.metrics.view;
+
+/** An {@code AttributesProcessor} is used by {@code View}s to define attribute processing. */
```

Comment:
Can we keep this class private? I think it's a bit early for it and shouldn't be needed with the configuration methods we have right now

Reply:
However, there would be another option if the `io.opentelemetry.sdk.metrics.view` package was collapsed into `io.opentelemetry.sdk.metrics`. If `SdkMeterProvider` was in the same package as `View`, then `View` could have package private methods for accessing configuration parameters, including any experimental features like appending baggage attributes.

Reply:
Hmm - yeah moving the code into the same package seems like a nice idea in general

Reply:
I've done this in my latest commit and think it makes sense. I've updated the PR title and description to reflect the current changes.